README.md

```diff
@@ -8,93 +8,95 @@
-  * FMUv5 (ARM Cortex M7)
-    * [Pixhawk 4](https://docs.px4.io/en/flight_controller/pixhawk4.html)
-    * [Pixhawk 4 mini](https://docs.px4.io/en/flight_controller/pixhawk4_mini.html)  
-  * [Airmind MindPX V2.8](http://www.mindpx.net/assets/accessories/UserGuide_MindPX.pdf)
-  * [Airmind MindRacer V1.2](http://mindpx.net/assets/accessories/mindracer_user_guide_v1.2.pdf)
-  * [Bitcraze Crazyflie 2.0](https://docs.px4.io/en/flight_controller/crazyflie2.html)
-  * [Omnibus F4 SD](https://docs.px4.io/en/flight_controller/omnibus_f4_sd.html)
-  * [BeagleBone Blue](https://docs.px4.io/en/flight_controller/beaglebone_blue.html)
-  * [Holybro Durandal](https://docs.px4.io/en/flight_controller/durandal.html)
-  * [Holybro Kakute F7](https://docs.px4.io/en/flight_controller/kakutef7.html)
-
-Additional information about supported hardware can be found in [PX4 user Guide > Autopilot Hardware](https://docs.px4.io/en/flight_controller/).
+
+* [Snapdragon Flight](https://docs.px4.io/en/flight_controller/snapdragon_flight.html)
+* [Intel Aero](https://docs.px4.io/en/flight_controller/intel_aero.html)
+* [Raspberry PI with Navio 2](https://docs.px4.io/en/flight_controller/raspberry_pi_navio2.html)
+* [Parrot Bebop 2](https://dev.px4.io/en/advanced/parrot_bebop.html)
+* FMUv2
+  * [Pixhawk](https://docs.px4.io/en/flight_controller/pixhawk.html)
+  * [Pixhawk Mini](https://docs.px4.io/en/flight_controller/pixhawk_mini.html)
+  * [Pixfalcon](https://docs.px4.io/en/flight_controller/pixfalcon.html)
+* FMUv3 
+  * [Pixhawk 2](https://docs.px4.io/en/flight_controller/pixhawk-2.html)
+  * [CUAV Pixhack v3](https://docs.px4.io/en/flight_controller/pixhack_v3.html)
+* FMUv4
+  * [Pixracer](https://docs.px4.io/en/flight_controller/pixracer.html)
+  * [Pixhawk 3 Pro](https://docs.px4.io/en/flight_controller/pixhawk3_pro.html)
+* FMUv5 \(ARM Cortex M7\)
+  * [Pixhawk 4](https://docs.px4.io/en/flight_controller/pixhawk4.html)
+  * [Pixhawk 4 mini](https://docs.px4.io/en/flight_controller/pixhawk4_mini.html)  
+  * [CUAV V5+](https://docs.px4.io/en/flight_controller/cuav_v5_plus.html)
+  * [CUAV V5 nano](https://docs.px4.io/en/flight_controller/cuav_v5_nano.html)
+* [Airmind MindPX V2.8](http://www.mindpx.net/assets/accessories/UserGuide_MindPX.pdf)
+* [Airmind MindRacer V1.2](http://mindpx.net/assets/accessories/mindracer_user_guide_v1.2.pdf)
+* [Bitcraze Crazyflie 2.0](https://docs.px4.io/en/flight_controller/crazyflie2.html)
+* [Omnibus F4 SD](https://docs.px4.io/en/flight_controller/omnibus_f4_sd.html)
+* [BeagleBone Blue](https://docs.px4.io/en/flight_controller/beaglebone_blue.html)
+* [Holybro Durandal](https://docs.px4.io/en/flight_controller/durandal.html)
+* [Holybro Kakute F7](https://docs.px4.io/en/flight_controller/kakutef7.html)
+
+Additional information about supported hardware can be found in [PX4 user Guide &gt; Autopilot Hardware](https://docs.px4.io/en/flight_controller/).
```

Comment:
The link should be https://docs.px4.io/master/en/flight_controller/

FWIW there is no need to use  `&gt;` in markdown unless you're inside an html block, and I wouldn't

Reply:
Why is the Holybro Durandal link correct? I want to use similar links from other flight controllers. What should I do? For the master may mean that it may be in beta, which is not what I want.